themes/jb/layouts/episode/single.html

```diff
@@ -126,6 +126,19 @@ <h2>Sponsors</h2>
               </div>
             </div>
           {{ end }}
+          <!-- Value for Value splits -->
+          {{ if .Params.value }}
+            <div class="tile is-child columns is-flex-grow-0" >
+              <div class="column px-0">
+                {{ with .Params.value.recipients }}
+                  <h3>Value for Value</h3>
```

Comment:
I think a better title for this would be `Boost Splits`

Reply:
Sounds good to me :+1: